samples/AppSecretsConfig/README.md

```diff
@@ -141,7 +141,7 @@ Next you'll need to add the App Configuration connection string from the templat
 2. With a *.cs* file open the command palette and run `Debug: Start debugging` or press `F5` (default binding).
 3. If prompted, select ".NET Core" to create a launch configuration and start debugging.
 
-#### [.NET CLI](#tab/dotnet_cli)
+#### [.NET](#tab/dotnet)
```

Comment:
This is great;  I didn't know it was a thing.

Reply:
Yeah - and turns out the values after "#tab" are of a fixed quantity: https://review.docs.microsoft.com/help/contribute/validation-ref/tabbed-conceptual?branch=master#approved-tabs